### PR TITLE
Super Admins can list unit users

### PR DESCRIPTION
**Pull request by i-oden**
Added unit option to `dds user ls` so that Super admins can specify a unit in which they would like to list users. 

**Test with:** https://github.com/ScilifelabDataCentre/dds_web/pull/1059

Before submitting a PR to the `dev` branch:
- [x] Tests passing
- [x] Black formatting
- [ - ] Rebase/merge the `dev` branch
- [x] Note in the CHANGELOG

Additional checks before submitting a PR to the `master` branch:
- Change version in `setup.py` (?)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -129,3 +129,4 @@ Please add a _short_ line describing the PR you make, if the PR implements a spe
 * When using `dds project access fix`, list the projects which where not possible to update access in ([#379](https://github.com/ScilifelabDataCentre/dds_cli/pull/379))
 * Add `Access` column to show user if they have access or not ([#383](https://github.com/ScilifelabDataCentre/dds_cli/pull/383))
 * New `--mount-dir` option for `dds data put` where the `DataDelivery...` folders will be created if specified ([#393](https://github.com/ScilifelabDataCentre/dds_cli/pull/393))
+* `--unit` option for Super Admins to list unit users ()
```
Comment (inaod568) [inline CHANGELOG.md:132]:
```suggestion
* `--unit` option for Super Admins to list unit users ([#397](https://github.com/ScilifelabDataCentre/dds_cli/pull/397))
```

File: dds_cli/options.py [hunk 2]
```diff
@@ -4,6 +4,9 @@
 import click
 from dds_cli.utils import multiple_help_text
 
+
+# Options
+
```
Comment (talavis) [inline dds_cli/options.py:9]: I don't see any reason to have this comment. We have the module doc string on top as well as the file name to tell us that this file contains options.
```suggestion
```

**Review by talavis — APPROVED** [review]
Seems to work as intended.

**Review by valyo — APPROVED** [review]
Looks like it works as expected and I nothing to complain about in the code

**Review by aanil — APPROVED** [review]
👍 Code looks ok to me!

**Pull request merged**